except_star.md

```diff
@@ -885,11 +885,22 @@ be attached to any exception by a `with_traceback()` call, it is hard to
 imagine a case where it makes sense to assign a traceback tree to an exception
 group.  Furthermore, a useful display of the traceback includes information
 about the nested exceptions. For this reason we decided it is best to leave
-the traceback mechanism as it is
-and modify the traceback display code.
+the traceback mechanism as it is and modify the traceback display code.
 
+### A full redesign of `except`
 
-### Adoption of try..except* syntax
+We considered introducing a new keyword (such as `catch`) which can be used
+to handle both plain exceptions and `ExceptionGroup`s. Its semantics would
+be the same as those of `except*` when catching an `ExceptionGroup`, but
+it would not wrap plain a exception to create an `ExceptionGroup`. This
+would have been part of a long term plan to replace `except` by `catch`,
+but we decided that deprecating `except` in favour of an improved keyword
+is too hard at this time, and it is more realistic to introduce the
```

Comment:
I'd replace "too hard" with "confusing for the users". I believe that `except *` makes it immediately apparent to Python coders that something involving multiple exception types will happen in this block. Whereas `catch` doesn't convey any of that meaning.

Moreover `catch` in Python would be confusing to JS and Java developers, for example. The regular exception catching syntax they accustomed to is `try...catch`, so they would most likely be confused which exception handling syntax they should use in Python. Even I, when I write too much JS code for too long and suddenly switch to Python make this mistake and type `try..catch`.